docs/usage/config-presets.md

```diff
@@ -42,6 +33,17 @@ Alternatively, Renovate can fetch preset files from an HTTP server.
 
 You can set a Git tag (like a SemVer) to use a specific release of your shared config.
 
+### Preset File Naming
+
+Presets are repo-hosted, and you can have one or more presets hosted per repository.
+If you omit a file name from your preset (e.g. `github>abc/foo`) then Renovate will look for a `default.json` file in the repo.
+If you wish to have an alternative file name, you need to specify it (e.g. `github>abc/foo:alternative-name.json5`).
```

Comment:
```suggestion
If you wish to have an alternative file name, you need to specify it (e.g. `github>abc/foo//alternative-name.json5`).
```
I think we need to use this path syntax for file extension presets

Reply:
yes, we should remove the subpresets stuff from docs to soft deprecating them

Reply:
It took a lot of effort to get those tables in there and correct. Please make sure we _really_ don't need them anymore, before dropping them. 😄

I know we can always get the tables back again from the Git commit history later, if we need, but that's extra work.